src/app/body/board/sprint-details/sprint-details.component.ts

```diff
@@ -21,6 +21,8 @@ import { Sprint } from 'src/app/Interface/TeamInterface';
 import { PopupHandlerService } from 'src/app/services/popup-handler/popup-handler.service';
 import { AuthService } from '../../../services/auth.service';
 
+import { LoaderComponent } from './../../loader/loader.component';
```

Comment:
Importing loader component in ts file is not necessary to implement the loader. Using it in html file is enough. Remove this